packages/dom/src/serialize-dom.js

```diff
@@ -35,7 +35,12 @@ export function serializeDOM(options) {
   let clone = dom.cloneNode(true);
   serializeInputs(dom, clone);
   serializeFrames(dom, clone, { enableJavaScript });
-  serializeVideos(dom, clone);
+
+  try {
+    serializeVideos(dom, clone);
+  } catch (error) {
+    console.log(`Couldn't serialize videos: ${error}`);
```

Comment:
Are we sure we want to log this? Could potentially cause some runners like karma or cypress to show these where the user won't know what it's referring to.

We should at least add `[percy]` to this if we want to keep it. But I vote for having a silent failure for now until we implement other serialization warnings and logs like planned.

Reply:
Also, since this try-catch is wrapped around the function invocation, this would result in any single failed video serialization to cause any remaining videos to not serialize since the loop would be broken. Instead, we should move this try-catch to within the `serializeVideos` loop to give each individual video a chance to serialize.